packages/docs/src/components/sidebar/Sidebar.vue

```diff
@@ -7,15 +7,17 @@
         v-for="(route, key) in navigationRoutes"
         :key="key"
       >
-        <template #header="{ value }">
-          <va-sidebar-item :activeColor="activeColor" @keydown.stop>
-            <va-sidebar-item-content :class="{ 'va-sidebar-item--active': isRouteHasActiveChild(route) }">
-              <va-sidebar-item-title>
-                {{ $t(route.displayName) }}
-              </va-sidebar-item-title>
-              <va-icon v-if="route.children" :name="value ? 'expand_less' : 'expand_more'" />
-            </va-sidebar-item-content>
-          </va-sidebar-item>
+        <template #header="{ value, hasKeyboardFocus }">
+          <div
```

Comment:
It would be great if you fix the issue with the VaSidebarItem and VaCollapse components instead of removing them from the docs page.

![image](https://user-images.githubusercontent.com/23530004/121717339-bc48a400-cae9-11eb-8af3-0f8a263cbdc6.png)

There are two ways:
- Set tabindex -1 for <a> and react to VaCollapse focus
- Remove tabindex for VaCollapse header, to prevent extra focusing.

Reply:
The thing here is that, as per @asvae , there should be no links (anchors) at all inside the va-collapse's header (and it actually makes sense since we don't want to redirect the user somewhere - instead, we only want to open the collapse). And since the va-sidebar-item implys the use of links (it uses the `<a>` tag), I have to stop using the va-sidebar-item for collapse's header at all.

Here's how the new markup looks like:
![Screenshot_20210614_182450](https://user-images.githubusercontent.com/9782236/121917557-e85c6300-cd3d-11eb-97b3-56318f233bf8.png)

Reply:
And yep, the collapse's header itself still needs to be `tabindex`able to make it possible to open/close collapses using the keyboard.

Reply:
@m0ksem I'm frankly not seeing any point jamming sidebar-item into collapse header. Outside of styling similarity it doesn't extend any of its behavior.